apps/pdf-service/src/puppeteer.spec.ts

```diff
@@ -52,6 +52,12 @@ describe('puppeteer', () => {
               header
             </p>
           </div>`,
+        additionalStyles: `<style>
+          div {
+            font-size: 32px;
```

Comment:
is this the default size that we are using across the pdf?

Reply:
No, it's just some arbitrary size for testing purposes

Reply:
oh!  my bad! its a spec file